net/teleport/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        gravitational teleport 2.0.6 v
+revision            0
+
+homepage            http://gravitational.com/teleport/
+categories          net
+maintainers         gclawes
+description         Teleport SSH server and Certificate Authority
+long_description    Teleport is a modern SSH server and CA for managing clusters \
+                    that provides secure login with certificate-based access, \
+                    2-factor authentication, session logging and replay, \
+                    integration with Google Apps and other OAuth identity providers, \
+                    and a Web UI.  Built on the Golang SSH library, and compatible with OpenSSH \
+license             Apache2
+
+checksums           rmd160  19d99162dbde0190ddadde27c855b9c21a716c8b \
+                    sha256 f6211d5a52d5b3806f44ae5ef3bda349d1c6924961ba70b953148ad9e793f3f1
+
+set goproj          github.com/${github.author}/${github.project}
+
+depends_lib         port:go port:zip
+platforms           darwin
+use_configure       no
+worksrcdir          src/${goproj}
+
+post-extract {
+    xinstall -d ${workpath}/src/github.com/${github.author}
+    move ${workpath}/${name}-${github.version} \
+        ${worksrcpath}
+}
+
+build {
+    system -W ${worksrcpath} "GOPATH=${workpath} make release"
```

Comment:
Using that, I get the error below.  It seems it needs the `-W ${worksrcpath}` argument to work build correctly (according to the [guide](https://guide.macports.org/#reference.phases.build) make runs in ${worksrcdir} by default)

```
:info:build make: Entering directory `/opt/local/var/macports/build/_Users_gclawes_development_macports-ports_net_teleport/teleport/work/src/github.com/gravitational/teleport'
:info:build rm -rf build
:info:build VERSION=2.0.6 /Applications/Xcode.app/Contents/Developer/usr/bin/make -f version.mk setver
:info:build make[1]: Entering directory `/opt/local/var/macports/build/_Users_gclawes_development_macports-ports_net_teleport/teleport/work/src/github.com/gravitational/teleport'
:info:build fatal: Not a git repository (or any of the parent directories): .git
:info:build make[1]: Leaving directory `/opt/local/var/macports/build/_Users_gclawes_development_macports-ports_net_teleport/teleport/work/src/github.com/gravitational/teleport'
:info:build rm -rf teleport
:info:build go build -o build/tsh -i  -ldflags '-w -s' ./tool/tsh
:info:build rm -rf *.gz
:info:build go build -o build/teleport -i  -ldflags '-w -s' ./tool/teleport
:info:build go build -o build/tctl -i  -ldflags '-w -s' ./tool/tctl
:info:build cd web/dist ; zip -qr ../../build/webassets.zip .
:info:build zip I/O error: No such file or directory
:info:build zip error: Could not create output file (../../build/webassets.zip)
:info:build make: *** [build/webassets.zip] Error 15
:info:build make: *** Waiting for unfinished jobs....
:info:build make: Leaving directory `/opt/local/var/macports/build/_Users_gclawes_development_macports-ports_net_teleport/teleport/work/src/github.com/gravitational/teleport'
:info:build Command failed:  cd "/opt/local/var/macports/build/_Users_gclawes_development_macports-ports_net_teleport/teleport/work/src/github.com/gravitational/teleport" && /usr/bin/make -j4 -w release
:info:build Exit code: 2
```